cmd/zetae2e/local/local.go

```diff
@@ -147,7 +144,19 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 	}
 
 	// initialize context
-	ctx, cancel := context.WithCancel(context.Background())
+	ctx, legacyCancel := context.WithTimeoutCause(context.Background(), TestTimeout, ErrTopLevelTimeout)
```

Comment:
Are `timeoutCancel` and `systemCancel` more explicit names?

Reply:
I can call it timeoutCancel it doesn't really matter to me